src/boot/test/boot-test.js

```diff
@@ -103,13 +104,31 @@ describe('bootstrap', () => {
       runBoot('annotator');
       const expectedAssets = [
         'scripts/annotator.bundle.1234.js#ts=123',
-        'styles/annotator.1234.css',
+        'styles/highlights.1234.css',
         'styles/pdfjs-overrides.1234.css',
       ].map(assetUrl);
 
       assert.deepEqual(findAssets(iframe.contentDocument), expectedAssets);
     });
 
+    it('preloads assets used wihin shadow roots in the annotation layer', () => {
+      runBoot('annotator');
+
+      const preloadLinks = [
+        ...iframe.contentDocument.querySelectorAll('link[rel=preload]'),
+      ];
+      preloadLinks.sort((a, b) => a.href.localeCompare(b.href));
```

Comment:
I think I copied and adapted this from somewhere else. This isn't strictly necessary but I think it makes sense in case we add other preloads in future.